src/lib/users/users.ts

```diff
@@ -28,7 +28,7 @@ export class Users {
         return res.data
     }
 
-    async update(id: string, user: User): Promise<User> {
+    async update(id: string, user: Partial<User>): Promise<User> {
         const { data: res } = await this.httpAgent.put(`users/${id}`, { data: user })
         return res.data
     }
```

Comment:
Only one thing, do you think this should not be a partial since it's a PUT request

Reply:
I tested against their API. It actually produces various errors if you send the whole user object.